cloud/openstack/os_keystone_service.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Hewlett-Packard Development Company, L.P.
+# Author: Davide Guerri <davide.guerri@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+try:
+    import shade
+
+    HAS_SHADE = True
+except ImportError:
+    HAS_SHADE = False
+
+DOCUMENTATION = '''
+---
+module: os_keystone_service
+version_added: "1.9"
+short_description: Manage OpenStack Identity (keystone) endpoints
+extends_documentation_fragment: openstack
+description:
+   - Manage endpoints from OpenStack.
+options:
+   name:
+     description:
+        - OpenStack service name (e.g. keystone)
+     required: true
+   service_type:
+     description:
+        - OpenStack service type (e.g. identity)
+     required: true
+   description:
+     description:
+        - Service description
+     required: false
+     default: Not provided
```

Comment:
Could we default to a description that matches the type? That's a tad more informative to end users.